src/Postnl.php

```diff
@@ -453,6 +453,21 @@ public function getDeliveryDate(
     }
 
     /**
+     * Returns location information of the supplied location code
+     *
+     * @param string $locationCode LocationCode information
+     * @param string $retailNetworkID PNPNL-01 is the code that can be used for all Dutch locations
```

Comment:
While `ID` (uppercase) is used commonly (including by PostNL), it's actually an abbreviation and **not** an acronym. It should therefore be `Id`.